tests/unit/htdocs/modules/system/SystemMaintenanceTest.php

```
@@ -261,4 +261,359 @@ public function isValidTableCachesResults(): void
+        }
+        foreach (scandir($scratchAbs) ?: [] as $entry) {
+            if ('.' === $entry || '..' === $entry) {
+                continue;
+            }
+            $path = $scratchAbs . '/' . $entry;
+            if (is_file($path)) {
+                @unlink($path);
+            }
+        }
+        @rmdir($scratchAbs);
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 348. (undefined)

(ErrorControlOperator)